cypress/support/commands.js

```diff
@@ -140,3 +140,70 @@ Cypress.Commands.add(
     );
   },
 );
+
+/**
```

Comment:
This is in this PR, but it is in #13151 with documentation, so merge that first.